cluster/gce/gci/configure-helper.sh

```diff
@@ -2399,6 +2399,22 @@ EOF
   fi
   if [[ "${ENABLE_METRICS_SERVER:-}" == "true" ]]; then
     setup-addon-manifests "addons" "metrics-server"
+    base_metrics_server_cpu="40m"
+    base_metrics_server_memory="40Mi"
+    metrics_server_memory_per_node="4"
```

Comment:
Same as above

Reply:
Can you please clarify which previous comment you're referring to?